server/execution/api/patient.py

```diff
@@ -87,3 +121,22 @@ def get_all_patient():
         }
     except KeyError:
         return "Missing or invalid request parameter detected.", 400
+
+
+def __try_leave_patient(execution_id: int, player: Player):
+    """ This method delays a location leave for a provided player. """
+    ptime.sleep(0.5)
```

Comment:
Why exactly 0.5?

Reply:
a second seemed to long for handling a request and everything below 0.4 may collide as race conditions with previous requests.